docs/src/components/productMaterial/MaterialHero.tsx

```diff
@@ -246,7 +246,7 @@ export default function MaterialHero() {
             make it easy to implement your own custom design system.
           </Typography>
           <GetStartedButtons
-            to="/material-ui/getting-started/"
+            primaryCTAto={ROUTES.materialDocs}
             installation="npm install @mui/material @emotion/react @emotion/styled"
           />
```

Comment:
It could be different, like 

<img width="517" alt="Screenshot 2023-08-03 at 00 49 04" src="https://github.com/mui/material-ui/assets/3165635/eff56e02-cd56-49f0-8a42-d2eaafec3e7f">

🤷‍♂️

Reply:
Not sure I understood what you're saying 😬

Reply:
I was curious, did a try, not sure how I feel about it.

Reply:
But what exactly? The button's label or anything else? 😅